src/views/Stake/StakeV2.js

```diff
@@ -912,160 +1302,321 @@ export default function StakeV2({
+                            selectedToken={CompatibleTokenEnum.MYC}
+                          />
+                          <StakeV2Styled.StakingButton onClick={() => stakeToken(MYC_TOKEN, UNSTAKE)}>
+                            Withdraw
+                          </StakeV2Styled.StakingButton>
+                        </StakeV2Styled.FlexRowEnd>
+                        <StakeV2Styled.FlexRowEnd>
+                          <WalletBalance
+                            walletAmount={userStakedEsMycBalance || ZERO_BN}
+                            tokenUsdPrice={mycPrice || ZERO_BN}
+                            selectedToken={CompatibleTokenEnum.esMYC}
+                          />
+                          <StakeV2Styled.StakingButton onClick={() => stakeToken(ES_MYC_TOKEN, UNSTAKE)}>
+                            Withdraw
+                          </StakeV2Styled.StakingButton>
+                        </StakeV2Styled.FlexRowEnd>
+                      </div>
+                    </div>
+                  </StakeV2Styled.StakingBannerRow>
+                  {stakingApr && (
+                    <StakeV2Styled.StakingBannerRow>
+                      <div className="App-card-row">
+                        <div className="label">APR</div>
+                        <div>
+                          <b>{stakingApr}%</b>
+                        </div>
+                      </div>
+                    </StakeV2Styled.StakingBannerRow>
+                  )}
+                  <StakeV2Styled.StakedTokens borderTop wrap>
+                    <StakeV2Styled.FlexRowBetween>
+                      <span>Claimable Rewards&nbsp;</span>
+                      <StakeV2Styled.FlexRowEnd>
+                        <WalletBalance
+                          large
+                          walletAmount={rewardsEarned || ZERO_BN}
+                          tokenUsdPrice={nativeTokenPrice || ZERO_BN}
+                          selectedToken={CompatibleTokenEnum.WETH}
+                          decimals={6}
+                        />
+                      </StakeV2Styled.FlexRowEnd>
+                    </StakeV2Styled.FlexRowBetween>
+                    <StakeV2Styled.ClaimButtonContainer>
+                      <StakeV2Styled.StakingButton onClick={claimRewards} fullWidth whiteText>
+                        Claim Rewards Now
+                      </StakeV2Styled.StakingButton>
+                    </StakeV2Styled.ClaimButtonContainer>
+                  </StakeV2Styled.StakedTokens>
+                  {cumulativeRewards && nativeTokenPrice && (
+                    <StakeV2Styled.StakingBannerRow className="total-rewards">
+                      <div className="App-card-row break">
+                        <div className="label">Total Rewards</div>
+                        <div>
+                          <span>
+                            <b>{convertBigNumberToString(cumulativeRewards, 6)} WETH</b>
```

Comment:
why 6 decimals here? or is that after the decimal place?

Reply:
after the decimal place - think that it could use formatAmount anyway, will have a look